src/ru/freeIt/homework/streamAPI/task2/Query2.java

```diff
@@ -0,0 +1,28 @@
+package ru.freeIt.homework.streamAPI.task2;
+
+import java.io.*;
+import java.util.Arrays;
+
+//Задать массив строк. Используя средства Stream API отсортировать строки в
+//        лексикографическом порядке по возрастанию и убыванию
+public class Query2 {
+    public static void main(String[] args) {
+        File file = new File("src/ru/freeIt/homework/streamAPI/task2/resource/string.txt");
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
+            StringBuilder stringBuilder = new StringBuilder();
+            while (bufferedReader.ready()) {
+                stringBuilder.append(bufferedReader.readLine());
+            }
+            bufferedReader.close();
+            String[] stringList = stringBuilder.toString().replaceAll("\\p{P}", "").split(" ");
+            Arrays.stream(stringList).sorted(String::compareToIgnoreCase).forEach(System.out::println);
```

Comment:
Try this:
Comparator.naturalOrder()

Reply:
sorry, but main idea it's using StreamAPI and lambda and I use it.

Reply:
You will use it. Final result should be:

```
Arrays.stream(stringList)
  .sorted(Comparator::naturalOrder)
  .forEach(System.out::println);
```